test/crowdsale/CapStagedCrowdsale.test.js

```diff
@@ -0,0 +1,69 @@
+const CapStagedCrowdsale = artifacts.require('CapStagedCrowdsale');
+const BigNumber = web3.BigNumber;
+const CapStagedCrowdsaleToken = artifacts.require('CapStagedCrowdsaleToken');
+require('chai').use(require('chai-as-promised')).use(require('chai-bignumber')(BigNumber)).should();
+
+contract('CapStagedCrowdsale', accounts => {
+  const _wallet = accounts[0];
+  const _rate = new BigNumber(1000);
+  const stage1Limit = 10000000000000000000; //  10ether
```

Comment:
likewise here, replace these large numbers with `web3.toWei`

Reply:
i've corrected this